src/iedit.cpp

```diff
@@ -2368,8 +2412,78 @@ void iedit_parse(struct descriptor_data * d, const char *arg)
             case CYB_HANDBLADE:
             case CYB_HANDSPUR:
             case CYB_REFLEXTRIGGER:
-            case CYB_SKULL:
-            case CYB_TORSO:
+            case CYB_CYBERARMS:
+              if (number < 0 || number > 2) {
+                send_to_char("Invalid Input! Enter options (0 to quit): ", CH);
+                return;
+              }
+              if (number == 0) {
+                iedit_disp_menu(d);
+                return;
+              }
+              number--;
+              TOGGLE_BIT(GET_CYBERWARE_FLAGS(OBJ), 10 << number);
```

Comment:
`10 << number` should be `1 << number` unless you're positive you want the input values to be 20 and 40

Reply:
Whoops, definitely should be a 1